src/v2/providers/alerts/billing.ts

```diff
@@ -32,7 +32,7 @@ export type BillingEvent<T> = CloudEvent<FirebaseAlertData<T>, WithAlertType>;
 /** @internal */
 export const planUpdateAlert = 'billing.planUpdate';
 /** @internal */
-export const automatedPlanUpdateAlert = 'billing.automatedPlanUpdate';
+export const planAutomatedUpdateAlert = 'billing.automatedPlanUpdate';
```

Comment:
huh this is bit odd. The event type is still `automatedPlanUpdate` not `planAutomatedUpdate`?

Reply:
Just double checked the alert type string, it's set to `billing.automatedPlanUpdate`. Since all their other methods and structures have this as planAutomatedUpdate, I'm going to request that they change this value on the provider end. I'll leave this PR up until it gets sorted out.

Reply:
What's in the spec? IDK if we should differ between the alerttype and the handler function name. It just seemed odd that this isn't object -> verb naming.

Reply:
@inlined @taeold I just submitted a change to the alerttype parameter to set the value to `billing.planAutomatedUpdate`